### ISSUE TITLE
The Top part of ViewController doesn't appear normally 

### ISSUE THREAD

**Issue by BasmaElias** [John-Lluch/SWRevealViewController#604]
i have a left menu and the front view controller display normally when the app run but if i open the menu and choose any item the top part of the UIViewController doesn't display normally .. if i rotate the device and return it the top part of UIViewController be normal .. 

![13645237_296957087309344_8254403250975511914_n](https://cloud.githubusercontent.com/assets/17385302/16730386/f142f94e-4772-11e6-818e-9dfdac370345.jpg)
![13669183_296957073976012_7265278545717353461_n](https://cloud.githubusercontent.com/assets/17385302/16730387/f145f112-4772-11e6-8dc5-9c221830544f.jpg)

and it is my code `if let secondViewController = self.storyboard?.instantiateViewControllerWithIdentifier("ProtectedPageViewController") as? ProtectedPageViewController {
           let navController = UINavigationController(rootViewController: secondViewController)
           navController.setViewControllers([secondViewController], animated:true)
           self.revealViewController().setFrontViewController(navController,animated:true)
           self.revealViewController().setFrontViewPosition(.Left, animated: true)
       }`

**Comment by iDevelopper** [comment]
#555 should help you.

Otherwise, #603

**Comment by BasmaElias** [comment]
Thank you, its working now !

**Comment by BasmaElias** [comment]
viewController displayed normally  but  self.revealViewController()  equal nil and the menu button doesn't work

**Comment by iDevelopper** [comment]
Have you configured in your storyboard the controller to the class RevealViewController?

**Comment by BasmaElias** [comment]
yes , when i run app self.revealViewController() equal value then if i press menu button and go the any viewcontroller self.revealViewController() be nil 

The code : 
`self.revealViewController().revealToggleAnimated(true)
        if let secondViewController = self.storyboard?.instantiateViewControllerWithIdentifier("ProtectedPageViewController") as? ProtectedPageViewController {
            let navController = UINavigationController(rootViewController: secondViewController)
            navController.setViewControllers([secondViewController], animated:true)
            self.revealViewController().setFrontViewController(navController,animated:true)
        }`
if i delete self.revealViewController().revealToggleAnimated(true) , button will be work but the viewcontroller will not display normally

**Comment by iDevelopper** [comment]
You have just to set a target and an action for this button:

class ViewController1: UIViewController {

```
@IBOutlet weak var menuButton: UIBarButtonItem!

override func viewDidLoad() {
    super.viewDidLoad()

    menuButton.target = revealViewController()
    menuButton.action = #selector(SWRevealViewController.revealToggle(_:))
}
```

**Comment by BasmaElias** [comment]
Thank you 😃  , its working now
